benchmarks/src/main/scala/scalaz/zio/lockfree/impls/package.scala

```diff
@@ -0,0 +1,12 @@
+package scalaz.zio.lockfree
+
+package object impls {
+  def queueByType(typ: String, capacity: Int): LockFreeQueue[Int] = typ match {
```

Comment:
How about changing `tpe` to a `sealed trait` with different `case object`, then you could have a `lookup` method that returns `Option[QueueType]`. I know it's just used for benchmarking.

Reply:
Sure!